### ISSUE TITLE
Responsive Layout problem

### ISSUE THREAD

**Issue by Aanisha** [CircuitVerse/CircuitVerse#195]
I am submitting an issue,with which I want to work too.The <body> tag is not fixed and responsive.The <body> tag is scrollable horizontally too and does not fit screen width for every device.

**Comment by armudgal** [comment]
Can you please submit more information regarding this. Screenshots would be recommended. Thanks

**Comment by Aanisha** [comment]
Once we scroll down the web elements get unevenly shifted towards right,and
the contents are not fixed horizontally while we scroll.Here I attached the
ScreenShot.

**Comment by armudgal** [comment]
Hi, I am still not able to understand where the issue actually is. As I mentioned please attach screenshots for better understanding or send a PR if the fix is a quick one.

**Comment by woswos** [comment]
I saw this issue only after I fixed responsibility issue with the footer (Pr #354). @Aanisha  I am sorry for accidentally taking what you were working on.

Addition to the responsive footer problem, I didn't understand what is going with the "Editing Project" page. Somehow the footer is inside of the div for the main content. The header bar is also there but not visible. I can see it when I check the source code.

<img width="1680" alt="Screen Shot 2019-04-06 at 4 32 21 AM" src="https://user-images.githubusercontent.com/10498874/55662529-03a9ba80-5825-11e9-95c9-a0b8a0d179ca.png">
